app/src/main/java/com/artemzin/qualitymatters/api/ChangeableBaseUrl.java

```diff
@@ -5,12 +5,11 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import okhttp3.HttpUrl;
-import retrofit2.BaseUrl;
 
 /**
  * Such implementation allows us easily change base url in the integration and functional tests!
  */
-public class ChangeableBaseUrl implements BaseUrl {
+public class ChangeableBaseUrl {
```

Comment:
Retrofit 2.0.0 drops the mutable base URL, so this class doesn't make sense any more and it'll break the tests.